js/src/redux/actions/documentData.js

```diff
@@ -0,0 +1,60 @@
+export const SET_DOCUMENT_DATA = "SET_DOCUMENT_DATA";
+export const SET_DOCUMENT_TEXT = "SET_DOCUMENT_TEXT";
+export const SET_DOCUMENT_EXCERPT = "SET_DOCUMENT_EXCERPT";
+export const SET_DOCUMENT_TITLE = "SET_DOCUMENT_TITLE";
+
+/**
+ * The set data action for the documentDataReducer.
+ *
+ * @param {Object} data The data from the document.
```

Comment:
I guess it would be nice it was specified what kind of data can be passed along ( It can be inferred from the surrounding actions, but specifically adding it would take away any possible confusion ).